addons/skin.estuary/xml/Includes_DialogSelect.xml

```diff
@@ -66,9 +66,9 @@
 					</control>
 					<control type="textbox">
 						<left>135</left>
-						<top>52</top>
+						<top>50</top>
 						<right>20</right>
-						<height>65</height>
+						<height>67</height>
```

Comment:
these skin changes were done because otherwise a autowrapped `Label2` would cut off the bottom of letters like `ygq` etc. on the 2nd line